tutorials/3d/fps_tutorial/part_five.rst

```diff
@@ -527,7 +527,7 @@ Then we get the center of the screen by dividing the current :ref:`Viewport <cla
 ``project_ray_origin`` and ``project_ray_normal`` from the camera. If you want to know more about how these functions work, see :ref:`Ray-casting <doc_ray-casting>`.
 
 Next we send the ray into the space state and see if it gets a result. We add the player and the knife's :ref:`Area <class_Area>` as two exceptions so the player cannot carry
-themselves or the knife's collision :ref:`Area <class_Area>`.
+herself/himself or the knife's collision :ref:`Area <class_Area>`.
```

Comment:
That's awkward IMO. "themselves" was fine, and well established nowadays as gender neutral singular reflexive pronoun.

This should probably be added to the docs writing guidelines if not in there yet.